app/src/main/java/com/example/keniphhomemaintenance/dwelling_screen/DwellingView.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.keniphhomemaintenance.dwelling_screen
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.example.keniphhomemaintenance.navigation.Screen
+
+@Composable
+fun DwellingScreen(navController: NavController) {
+    var dwellingName by remember {
+        mutableStateOf("")
+    }
+
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(50.dp, 300.dp)
```

Comment:
dimens.xml or constants for numbers like this

Reply:
This was in @NickPerozzi's Screen so I didn't bother having the dimensions be in dimens.xml or constants since it would be changed later by Nick anyways